packages/remediations/src/steps/IssueResolutionStep.scss

```diff
@@ -1,5 +1,33 @@
 @import '~@redhat-cloud-services/frontend-components-utilities/files/Utilities/_all';
 
-.ins-c-resolution-option .pf-c-check{
-    align-items: baseline;
+.ins-c-resolution-container {
+    display: flex;
+    flex-wrap: wrap;
 }
+.ins-c-resolution-option{
+    width: 50%;
+    padding: 8px;
+    .pf-c-tile {
+        width: 100%;
+        height: 100%;
+        text-align: left;
+        .pf-c-tile__header {
+            justify-content: left;
+        }
+        .pf-c-tile__body {
+            display: flex;
+            .pf-c-content {
+                margin-top: auto;
+            }
+        }
+    }
+    .ins-c-reboot-required {
+        color: #826110
```

Comment:
Is there some PF variable that matches this color?

Reply:
I believe there is not.

Reply:
Wait. This might be it: --pf-global--warning-color--200